dotcom-rendering/src/web/browser/debug/README.md

```diff
@@ -0,0 +1,30 @@
+# Debug bookmarklet
+
+> Your chance to to see the website through DCRs eyes!
+
+The DCR debug bookmarklet is a script designed to be executed from a bookmarklet which will annotate a DCR page with debug information.
+
+Add the follow script string as the 'url' to your bookmark:
+
+```js
+javascript: (() => {
+    document.body.appendChild(document.createElement('script')).src =
+        'https://assets.guim.co.uk/assets/debug.js';
+})();
+```
+
+## Running locally
+
+If you need to run this locally, you can simply get it from the dev server:
+
+```js
+javascript: (() => {
+    document.body.appendChild(document.createElement('script')).src =
+        'http://localhost:3030/assets/debug.js';
+})();
+```
+
+## After publishing
+
+Assets on assets.guim.co.uk are cached by fastly - to allow users to receive the latest update of this bookmarklet,
```

Comment:
```suggestion
Assets on assets.guim.co.uk are cached by fastly. To allow users to receive the latest version of this bookmarklet after an update,
```

Reply:
I tripped up slightly on the original phrasing, as I thought it was saying that it was cached in order to allow users to receive the latest updates. This might just be me though!